mat64/dense.go

```diff
@@ -273,6 +297,9 @@ func (m *Dense) Reset() {
 	m.mat.Data = m.mat.Data[:0]
 }
 
+// Clone makes a copy of a into the receiver, overwriting the previous value of the receiver.
```

Comment:
I feel like the documentation for Clone/Copy should highlight the differences of behavior in Size. @kortschak ?